tests/test_core.py

```diff
@@ -269,6 +268,7 @@ def test_marc21_records_over_single_line():
 
 def test_records_marc21_tojson_tomarc21():
     """Test records marc21 - json - marc21."""
+    import json
```

Comment:
- similar to tests/test_utils.py
- IMHO we can make the imports global. It's probably because some of the first versions were doing nasty thing during import. My point was only about `import json`.